### ISSUE TITLE
IPHOD transcriptions?

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#441]
Do we still have an IPA-version of the IPHOD corpus? I just re-downloaded it and realized it's in arpabet. Might be nice to offer the option if we still have both?

**Comment by mmcauliffe** [comment]
I would argue against having IPHOD in IPA, since that was more of a stop gap when we didn't have features for arpabet.  The IPA was generated by a script, and I can't remember if we did anything beyond one-to-one hacks.  For instance, button in IPHOD should be something like "B.AH1.T.AH0.N" which would then get rendered in IPA as "b.ʌ.t.ə.n", where the final syllable should really be a syllabic nasal.  They have syllabic r (er), but no other syllabic consonants. Also, they have things like "HH.W.IH.CH" (which, because they don't have a voiceless w phoneme), which would then be rendered in IPA as "h.w.ɪ.tʃ".

My main issue with this is that the CMU dictionary is not a great dictionary for English, it just happens to be easily usable.  I'd rather not give arpabet a greater veneer of legitimacy by putting it into the IPA. I would vote instead for trying to find a better (pronunciation) dictionary for English and use subtlex frequencies with that.   I feel like we distributed an IPA version, we'd be a little bit culpable for the drawbacks of arpabet.
